packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/prepare-app/index.md

```diff
@@ -49,18 +49,18 @@ Select the type of app you want to create, choosing either **SWA** or **SAML 2.0
 1. Under **Supported provisioning actions**, choose the provisioning actions supported by your SCIM server.
 
     - Import New Users and Profile Updates: This option populates the **Settings > To Okta** page. You can specify the details of how Okta imports new users and user profile updates.
-    - Push New Users: This option populates the **Settings > To App** page, and contains settings for all the user information that flows from Okta into an app.
-    - Push Profile Updates: This option populates the **Settings > To App** page, and contains settings for all profile information that flows from Okta into an app.
-    - Push Groups: This option populates the Settings > To App page, and contains settings for all group information that flows from Okta into an app.
+    - Push New Users: This option populates the **Settings > To App** page, and contains settings for all the user information that flows from Okta into an application.
+    - Push Profile Updates: This option populates the **Settings > To App** page, and contains settings for all profile information that flows from Okta into an application.
+    - Push Groups: This option populates the Settings > To App page, and contains settings for all group information that flows from Okta into an application.
 
-1. In the Authentication Mode section, you can choose which mode you want to use for Okta to connect to your SCIM app.
+1. In the Authentication Mode section, you can choose which mode you want to use for Okta to connect to your SCIM application.
 
     - Basic Auth: To authenticate using Basic Auth mode, you need to provide the username and password for the account that handles the create, update, and deprovisioning actions on your SCIM server.
-    - HTTP Header: To authenticate using HTTP Header, you need to provide a bearer token that will provide authorization against your SCIM app. See [Create an API token](/docs/guides/create-an-api-token/) for instructions on how to generate a token.
+    - HTTP Header: To authenticate using HTTP Header, you need to provide a bearer token that will provide authorization against your SCIM application. See [Create an API token](/docs/guides/create-an-api-token/) for instructions on how to generate a token.
```

Comment:
All changes taken, except I re-wrote the sentence a bit to avoid the "provide X that provides Y"
> HTTP Header, enter a bearer token to provide authorization